plugins/check_dig.c

```diff
@@ -150,7 +151,7 @@ main (int argc, char **argv)
         }
 
         t = tt; /* consider the right-side token, does it match ex? */
-        if ( (strcasestr( t, ex ) == t) && (strlen( t ) == strlen( ex )) ) {
+        if ( (!exact && strcasestr( t, ex )) || ((strcasestr( t, ex ) == t) && (strlen( t ) == strlen( ex ))) ) {
```

Comment:
I agree with what Jason told you this morning - this conditional is kind of annoying to read, and I feel that you can replace the right side of the OR with one call to `strncmp()`

Reply:
Sure. My reasoning for this method was to minimally change the existing code. I have no qualms.